pandas/core/indexes/base.py

```diff
@@ -2412,7 +2412,7 @@ def isna(self) -> np.ndarray:
 
         Returns
         -------
-        numpy.ndarray
+        numpy.ndarray[bool]
```

Comment:
completely minor but i think inconsistent as use of `numpy.ndarray` vs `np.darray` (see this much more) vs `ndarray`

Reply:
i think you're right, decided against trying to make these uniform in this PR

Reply:
green

Reply:
yep next one (and prob should have a rule for this). note that I am saying it out loud but not knowing how hard that is :->